src/cdk/tree/tree-redesign.spec.ts

```diff
@@ -0,0 +1,1742 @@
+/**
```

Comment:
Feel free to create a temporary build rule just for these tests if that would make development easier

Reply:
I think it'll be fine as-is